scripts/analysis.py

```diff
@@ -1091,32 +1034,229 @@ def plot_power(cursor):
                       'Number of Reactors vs Time',
                       'number_plot', init_year)
 
+
+def plot_in_out_flux(cursor, facility, influx_bool, title, outputname):
+    """plots timeseries outflux from facility name in kg.
```

Comment:
That's an outdated documentaion. We must have missed it...
I'll fix it, thank you for letting us know.